INSTALL.md

```diff
@@ -3,19 +3,16 @@
 ## Prerequisites
 
 ### Hardware
-1. A Raspberry Pi or similar SBC capable of running Linux, with Bluetooth antenna.
-2. Micro SD card + SD adapter + necessary cables for OS installation (not detailed in this procedure).
-
-
-> [!Important]
-> In what follows, "RPi" refers to the device where the service will be executed.
+1. A Linux based computer which could be the same machine that runs Home Assistant (HA), or a seperate device located remotely from the HA machine, but close to where the car is located. We'll call this device the Host Device (HD). A Raspberry Pi makes an ideal HD

```

Comment:
I kind of dislike to tell people "close to where the car is" because we don't mention "what is considered close" and we lack to mention also that no BLE devices are made the same.

Are we talking about BLE 4.2 (60m) or 5.0 (240m)?  Perhaps in this Install and the add-on one we should mention people that if they are far to consider getting a 5.0 adapter prior to get a RPi or something similar just for that.

Also a 5.0 adapter connected directly to a USB port will have interference due to the USB port - an USB extension cord is required.

Not sure at all how we want to put that in here or elsewhere, my mind is a busy coding ;)